### ISSUE TITLE
Allow adding books from the list search

### ISSUE THREAD

**Issue by cincodenada** [bookwyrm-social/bookwyrm#1862]
**Is your feature request related to a problem? Please describe.**
I went to add a book to a list that wasn't yet present on my instance, and there was no option to make a wider search/import the book. It would be nice to be able to add books directly from that search box, rather than having to go through and import each book before adding it.

**Describe the solution you'd like**
Optimally, if the book isn't found locally, we'd kick off an OpenLibrary search automatically, like we do with the main search box.

**Describe alternatives you've considered**
It would also be fine to have a button to manually make a wider search, I think.

**Additional context**
In this case I'm trying to make a list based on an external source (Alok V. Menon's book reports), where I'm pretty sure none of the books on there are gonna be on my instance already - so having to add books separately basically doubles the amount of effort required to make this list.

**Comment by mouse-reeve** [comment]
Yeah! The thing that will make this a little complicated is that it will combine two things that are usually separate steps (resolving a book search result and adding a book to a list) into two steps, but that certainly can be done.
